exporter/test_exporter.py

```diff
@@ -1,17 +1,21 @@
-from unittest.mock import Mock, patch
-
-import google.cloud.exceptions
+from unittest import mock
+from unittest.mock import Mock
 import pytest
 import os
-
+import google.cloud.exceptions
 from flask.testing import FlaskClient
-from google.cloud import bigquery
+from google.cloud import bigquery  # type: ignore
+import pandas as pd
+
+from main import app, STATE_LEVEL_FIPS_LIST, get_table_name
 
-from main import app
+NUM_STATES_AND_TERRITORIES = len(STATE_LEVEL_FIPS_LIST)
 
-test_tables = [bigquery.Table("my-project.my-dataset.t1"),
-               bigquery.Table("my-project.my-dataset.t2_std"),
-               bigquery.Table("my-project.my-dataset.t3")]
+test_tables = [bigquery.Table("my-project.my-dataset.t1-sex"),
```

Comment:
Global constants should be ALL_CAPS. Otherwise I'm assuming its defined in the function.